src/app/test-record/emission-details/emission-details-edit/emission-details-edit.component.html

```diff
@@ -0,0 +1,143 @@
+<div class="govuk-grid-row">
```

Comment:
folder name to just `edit` instead of `emission-detail-edit`. Let have a uniform approach for the entire app. Please update all necessary editable folder